docs/UpstreamSupport.md

```diff
@@ -8,6 +8,39 @@ The Pelorus engineering team will provide `best-effort` level support for Peloru
 
 * You may also consider opening a [discussion](https://github.com/konveyor/pelorus/discussions) thread to discuss any feature, bug, or enhancement prior to opening a Github issue.
 
+## Backend Exporters
+
+Each of the three Pelorus exporters, the committime, deploytime and the failure exporter support various backends or services.  The Pelorus team is working to ensure an optimal experience for each backend service that is integrated with Pelorus.  Our goal is have each backend integrated into OpenShift's Prow Continuous Integration system to help ensure the quality of the integration of the service and Pelorus.
```

Comment:
Might I suggest a more minimalist version:

> Pelorus's exporters support various backends or services. The Pelorus team is working to ensure an optimal experience for each backend integrated with Pelorus. Our goal is for each backend to be checked by OpenShift's Prow Continuous Integration system, to ensure the quality of each service's integration with Pelorus.

Reply:
k.. thanks for the simplification ++ added